### PR TITLE
Add handlers for textfields and animation for move when keyboard presents and hides

### PR DESCRIPTION
**Pull request by pablomarti**
This PR:
- Adds handlers for textfields
- Adds animations for move the content of the card when the keyboard appears or disappears 
- Adds the dismiss listeners to the card
- Adds the function for hiding the card when the event related to the dismiss listener is dispatched

### CODE REVIEW

**Comment by ryancarlson** [comment]
It's not working for the original KGP either

**Comment by pablomarti** [comment]
Hey @ryancarlson the thing that didn't work for you is the **dismiss-listeners** and/or the keyboard?

**Comment by pablomarti** [comment]
@ryancarlson currently the XML is not returning the attribute `dismiss-listeners` in the `card` component

File: godtools/Views/TractElements/TractCard.swift [hunk 1]
```diff
@@ -113,6 +113,11 @@ class TractCard: BaseTractElement {
         return properties.listeners == "" ? nil : properties.listeners.components(separatedBy: " ")
     }
     
+    override func elementDismissListeners() -> [String]? {
+        let properties = cardProperties()
+        return properties.dismissListeners == "" ? nil : properties.dismissListeners.components(separatedBy: ",")
```
Comment (ryancarlson) [inline godtools/Views/TractElements/TractCard.swift]: @pablomarti i'll change this to space separated too.

**Pull request merged**